Package.swift

```diff
@@ -1,52 +1,33 @@
-// swift-tools-version:5.0
+// swift-tools-version:5.5
 import PackageDescription
 
 let package = Package(
     name: "swift-distributed-tracing-baggage",
+    platforms: [
+        .macOS(.v12),
+        .iOS(.v15),
+        .watchOS(.v8),
+        .tvOS(.v15),
+    ],
     products: [
         .library(
-            name: "Baggage",
+            name: "BaggageModule",
             targets: [
-                "Baggage",
+                "BaggageModule",
```

Comment:
👍

Reply:
hmm, this I am not sure about. I know there is previous art in using the `Module` suffix but its pretty leaky. what alternatives did we consider here?

Reply:
The alternatives are the long ones: `DistributedTracingBaggage` perhaps.
I figured that we can follow precendent set by stdlib where they often do `...Module`.

Maybe the long name could work?

Reply:
How about `InstrumentationBaggage`? With our current design it's tight to an `Instrument` (which might be a `Tracer`) so to me that name makes sense.

Reply:
That makes sense as well 👍